website/docs/plugin/sdkv2/logging/index.mdx

```diff
@@ -6,20 +6,20 @@ description: |-
 
 # Logging
 
-Terraform Plugin SDKv2 integrates with the structured logging framework [terraform-plugin-log](/plugin/log). High-quality logs are critical to quickly [debugging your provider](/plugin/debugging).
+Terraform Plugin SDKv2 integrates with the structured logging framework [terraform-plugin-log](/terraform/plugin/log). High-quality logs are critical to quickly [debugging your provider](/terraform/plugin/debugging).
 
 ## Managing Log Output
 
-Learn how to use [environment variables and other methods](/plugin/log/managing) to enable and filter logs.
+Learn how to use [environment variables and other methods](/terraform/plugin/log/managing) to enable and filter logs.
 
 ## Writing Log Output
 
-Learn how to [implement code in provider logic](/plugin/log/writing) to output logs.
+Learn how to [implement code in provider logic](/terraform/plugin/log/writing) to output logs.
 
 ## Filtering Log Output
 
-Learn how to [implement code in provider logic](/plugin/log/filtering) to omit logs or mask specific log messages and structured log fields.
+Learn how to [implement code in provider logic](/terraform/plugin/log/filtering) to omit logs or mask specific log messages and structured log fields.
 
 ## Log HTTP Transactions
 
-Learn how to [set up the Logging HTTP Transport](./logging/http-transport) to log HTTP Transactions with the [structured logging framework](/plugin/log).
+Learn how to [set up the Logging HTTP Transport](/terraform/plugin/sdkv2/logging/logging/http-transport) to log HTTP Transactions with the [structured logging framework](/terraform/plugin/log).
```

Comment:
🔴 `"set up the Logging HTTP Transport"` link is not correct

Reply:
This `dev-portal` PR fixes the issue identified: [[preAdjustUrl] Handle with currentPath has a trailing slash](https://github.com/hashicorp/dev-portal/pull/1615)

Reply:
Link for `[set up the Logging HTTP Transport]` should be `/terraform/plugin/sdkv2/logging/http-transport`